pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -442,7 +441,8 @@ message LinuxContainerSecurityContext {
     // SELinux context to be optionally applied.
     optional SELinuxOption selinux_options = 4;
     // The user to run the the container process as. Can be either UID or user
-    // name.
+    // name. If it's user name, it MUST exist in the container image, or else
+    // runtime MUST return an error.
```

Comment:
I think we can have both `run_as_user` (int64) and `run_as_username` (string), and be clear that only one of them will be set. We can deprecate `run_as_username` later. 

WDYT? @jonboulle @timstclair @feiskyer